### PR TITLE
(grc_conversion) continued EpiDoc and CTS conversion

### PR DESCRIPTION
**Pull request by lcerrato**
See #1399

### CODE REVIEW

File: data/tlg0086/tlg009/__cts__.xml
Comment (AlisonBabeu) [inline data/tlg0086/tlg009/__cts__.xml:1]: I think you want to add (printing) after the date here and in the other cts_xml file as well?

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
hi @lcerrato just a minor change to the dates (add printing) but otherwise looks good.

**Comment by lcerrato** [comment]
> hi @lcerrato just a minor change to the dates (add printing) but otherwise looks good.

I think I have them all?

**Review by gregorycrane — APPROVED** [review]

**Comment by gregorycrane** [comment]
A simple one character correction.

**Comment by lcerrato** [comment]
@gregorycrane 
Thanks—was waiting on Alison to approve the changes she requested earlier. I only appended this typo change to avoid switching branches because I have a large work in progress.

**Comment by lcerrato** [comment]
@AlisonBabeu 
Can you still take a look at the things you requested? I can add/change in the next PR.

**Pull request merged**